lib/vrl/parser/src/lex.rs

```diff
@@ -1695,7 +1695,7 @@ mod test {
     }
 
     #[test]
-    fn ampersat_in_query() {
```

Comment:
This is not a spelling mistake: https://www.thefreedictionary.com/Ampersat

Reply:
Doh - good catch, I've never heard of that term before!

Reply:
Would you be opposed to `at_sign_in_query`?

Reply:
I'd give `at_sign_in_query` a +1 given that google, dictionary, and merriam-webster don't recognize `ampersat` as a word...